cpp/test/prims/batched/batched_matrix.h

```diff
@@ -17,7 +17,8 @@
 #pragma once
 
 namespace MLCommon {
-namespace Matrix {
+namespace LinAlg {
+namespace Naive {
```

Comment:
maybe `MLCommon::LinAlg::Batched::Naive` is a better namespace for this?

Reply:
I'm not sure, as these operations aren't batched at all. They're used to test batched matrix operations but the naive version is called in a loop. Maybe I should move that up in the hierarchy and rename the file `linalg_naive.h`?

Reply:
Ok. That makes sense.